app/style/common/variables.less

```diff
@@ -18,11 +18,19 @@
  */
 
 // ----------------------------------------------------------------------------
+// Local Emoji Font
+// ----------------------------------------------------------------------------
+@font-face {
+					font-family: emoji;
+					src: local('Apple Color Emoji'), local('Android Emoji'), local('Segoe UI Emoji'), local('Segoe UI Symbol');
```

Comment:
Changes pushed! I assume you meant two actual spaces; if you meant tabs let me know.